docs/developer-guides/setting-up-token-management-partner-chains.md

```diff
@@ -0,0 +1,402 @@
+# Setting up token management for a partner chain
```

Comment:
I don't like this name. To avoid lengthy discussion here. I'll reply in a Slack thread.

Reply:
Going with @alexromanov's suggestion, I'm changing the name to "Native Token Reserve Management"